dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/TracingInterceptor.kt

```diff
@@ -82,11 +84,22 @@ internal constructor(
      * @param tracedRequestListener a listener for automatically created [Span]s
      */
     @JvmOverloads
+    @Deprecated("hosts should be defined in the DatadogConfig.setFirstPartyHosts()")
     constructor(
         tracedHosts: List<String>,
         tracedRequestListener: TracedRequestListener = NoOpTracedRequestListener()
     ) : this(tracedHosts, tracedRequestListener, { AndroidTracer.Builder().build() })
 
+    /**
+     * Creates a [TracingInterceptor] to automatically create a trace around OkHttp [Request]s.
+     *
+     * @param tracedRequestListener a listener for automatically created [Span]s
+     */
+    @JvmOverloads
+    constructor(
+        tracedRequestListener: TracedRequestListener = NoOpTracedRequestListener()
+    ) : this(emptyList(), tracedRequestListener, { AndroidTracer.Builder().build() })
```

Comment:
As this configuration is now less obvious (previously the hosts were expected in Interceptor's initializer, now they are part of the `Datadog.Config`), maybe it's worth printing console warning from here on misconfiguration? I mean: when the interceptor is created, but the `setFirstPartyHosts()` was not called.

Reply:
Indeed that's a good call.